docs/components/store.md

```diff
@@ -6,7 +6,7 @@ menu: components
 
 # Store
 
-The store component of Thanos implements the Store API on top of historical data in an object storage bucket. It acts primarily as an API gateway and therefore does not need significant amounts of local disk space. It joins a Thanos cluster on startup and advertises the data it can access.
+The `thanost store` component (also known as Store Gateway) of Thanos implements the Store API on top of historical data in an object storage bucket. It acts primarily as an API gateway and therefore does not need significant amounts of local disk space. It joins a Thanos cluster on startup and advertises the data it can access.
```

Comment:
```suggestion
The `thanos store` command of Thanos runs a component (also known as Store Gateway) that implements the Store API on top of historical data in an object storage bucket. It acts primarily as an API gateway and therefore does not need significant amounts of local disk space. It joins a Thanos cluster on startup and advertises the data it can access.
```